src/contentmanager.cpp

```diff
@@ -26,6 +26,7 @@ ContentManager::ContentManager(Library* library, kiwix::Downloader* downloader,
     connect(mp_library, &Library::booksChanged, this, [=]() {emit(this->booksChanged());});
     connect(this, &ContentManager::filterParamsChanged, this, &ContentManager::updateLibrary);
     connect(&m_remoteLibraryManager, &RemoteLibraryManager::booksChanged, this, &ContentManager::updateRemoteLibrary);
+    connect(&m_remoteLibraryManager, &RemoteLibraryManager::booksChanged, this, [=]() {emit(this->displayLoadIcon(false));});
```

Comment:
Could this signal being emit in `ContentManager::updateRemoteLibrary` instead of creating a new connection ?